python/private/toolchains_repo.bzl

```diff
@@ -150,27 +152,41 @@ def _toolchain_aliases_impl(rctx):
     is_windows = (os_name == WINDOWS_NAME)
     python3_binary_path = "python.exe" if is_windows else "bin/python3"
 
+    # uncertain if we need to do this check or not
+    full_python_version = rctx.attr.python_version
+    if TOOL_VERSIONS.get(full_python_version) == None:
+        full_python_version = MINOR_MAPPING[full_python_version]
```

Comment:
I had code in #1294 that was similar to this, it was called `full_version`, which would always ensure that the python version is in `X.Y.Z` format. I think we could cherry pick it from there. I did the cherry pick in #1340, but that has not been reviewed yet.

Reply:
And a nit: this could be changed to be `full_python_version = MINOR_MAPPING.get(rctx.attr.python_version, rctx.attr.python_version)` as we should have an entry in `MINOR_MAPPING` for each entry in `TOOL_VERSIONS`.

Reply:
I will wait for #1340 to get in and use that.